examples/cord_epsim/Makefile

```diff
@@ -35,13 +35,20 @@ CFLAGS += -DDEVELHELP
 # For debugging and demonstration purposes, we limit the lifetime to the minimal
 # allowed value of 60s (see draft-ietf-core-resource-directory-11, Table 2)
 RD_LT ?= 60
-# Override this variable to set the RD server address (default is the all nodes
-# multicast address)
-RD_ADDR ?= \"ff02::1\"
+CFLAGS += -DCORD_LT=$(RD_LT)
+
+# If nothing else is defined, we use CoAP default port
 RD_PORT ?= 5683
+CFLAGS += -DCORD_SERVER_PORT=$(RD_PORT)
 
-CFLAGS += -DCORD_LT=$(RD_LT)
+# The RD servers address must be defined by the build environment (by setting
+# the RD_ADDR environment variable. Per default, this value to the loopback
+# address for enabling the build tests to successfully build this example.
+RD_ADDR ?= \"::1\"
+ifeq (\"::1\", $(RD_ADDR))
+  $(info WARNING: you did not specify a valid RD server address)
```

Comment:
Yes, its true that the warning on a 'make clean' is not optimal. But using your proposed additional if is also not quite optimal, as it would not display the warning on the typical `make clean all` command.

Will think of something.